src/components/catigories.js

```diff
@@ -1,7 +1,11 @@
+import { useDispatch } from 'react-redux';
+import { underConstruction } from '../redux/categories/categories';
+
 function Categories() {
```

Comment:
- [ ] [OPTIONAL] Your file name has a typo (catigories) kindly consider correcting it

Reply:
Done 👍🏻